local/modules/TheliaSmarty/Template/Plugins/DataAccessFunctions.php

```diff
@@ -808,6 +928,8 @@ public function getPluginDescriptors()
             new SmartyPluginDescriptor('function', 'meta', $this, 'metaAccess'),
             new SmartyPluginDescriptor('function', 'module_config', $this, 'moduleConfigDataAccess'),
             new SmartyPluginDescriptor('function', 'coupon', $this, 'couponDataAccess'),
+            new SmartyPluginDescriptor('function', 'favicon', $this, 'faviconDataAccess'),
+            new SmartyPluginDescriptor('function', 'logo', $this, 'logoDataAccess'),
```

Comment:
Instead of creating two functions `logo` and `favicon`, a `template` function with an argument would be more future-proof :

`{template attr="logo"}` and `{template attr="favicon"}`